src/IW/Server/Issue.hs

```diff
@@ -27,11 +27,11 @@ issueServer = IssueSite
     { issuesRoute = issuesHandler
     }
 
-issuesHandler 
+issuesHandler
     :: ( WithDb env m
        )
     => Maybe Text
     -> m [WithId Issue]
 issuesHandler maybeLabel = case maybeLabel of
     Nothing    -> getIssues
-    Just label -> getIssuesByLabel label 
+    Just label -> undefined
```

Comment:
Sorry, is `getIssuesByLabel` not suiting here anymore?

Reply:
@vrom911 No it is I just have to modify the function because it’s a list and not just a single label anymore. I’m doing that now.